README.md

```diff
@@ -1,89 +1,494 @@
++ ***Contribute*** to a dwyl project
 or tutorial(s): https://github.com/dwyl?utf8=%E2%9C%93&q=learn
 (_a Pull Request with an improvement to the learning materials_)
-+ Answer the "AMA" questions see: `/ama`
+See: What is a "_Previous Contribution_" question/answer below.
 
-<br /><br /><br />
+### _Apply_!
 
-## Way More Questions & Answers!
+Send us an email: **hello@dwyl.com** with the subject: "Apprenticeship".
+Describe _why_ you want to join our Apprenticeship program and
+include a link to your GitHub profile.
+We will take it from there.
 
-### Q: What is a "Previous Contribution" to a @dwyl Project?
+<br /><br /><br />
+
+## Frequently Asked Questions & _Answers_!
 
-Contribution to a dwyl project can range from:
-+ asking a question (_seeking clarification_) in a GitHub Issue
-+ reporting a bug/issue on one of our projects or tutorials
-+ Creating a Pull Request that _fixes_ the bug/issue you reported
-and or adds the question/answer to a relevant readme.
 
 ### Q: How long is a day?
 [asked by @ZooeyMiller in #2](https://github.com/dwyl/apprenticeship/issues/2)
 
-Aim to start at 09:00. If you are prefer to start earlier,
-just let someone know you are available/online!
+You should aim to spend **6 hours** per day on _deep work_.<sup>1</sup>
+You are responsible for managing/tracking your time.
+Divide your time between learning, researching, writing, coding, editing
+and other creative work.
+Aim to **start** your day no later than **08:50 GMT**
+so that you are always "on time"
+for the daily morning meeting.
+
+If you are prefer to start earlier, do it!
+Some of us start our day at 06:00 so we get 3h
+of focus time *before* 09:00!
+Just let someone know you are available/online
+and remember to track your time so you don't over-work.
+
+Six hours either sounds like _little_
+(_if you are used to working 12h per day at a minimum wage job_),
+or a _lot_ if you have attempted to _focus_ without distractions for 6 hours.
+You are _not_ expected to work _continuously_ for 6 hours.
+You _are_ expected to keep _track_ of your time
+
+> <sup>1</sup>If you haven't yet read the book **Deep Work** by Cal Newport
+make it a priority to read it and understand all the concepts
+***before*** starting the ACT.
+
+
+### Q: How to _Communicate_?
+
+
+1. **GitHub** - GitHub is our
+[single source of truth](https://en.wikipedia.org/wiki/Single_source_of_truth).
+We have a _strong_ preference for communicating via GitHub.
+This ensures that _anyone_ can see what everyone _else_ is learning
+or working on and can _help_ when we get stuck.
+If in doubt, open an issue in the repository you are working on.
+
+
+2. Daily Standup Meetings - We have a
+[daily standup meeting](https://en.wikipedia.org/wiki/Stand-up_meeting)
+at **09:00** each day of the work week (_Monday - Friday_) <br />
+_Everyone_ attends via **Zoom** video conference:
+https://zoom.us/j/3371337133 <br />
```

Comment:
This will allow literally anyone to attend. Should be fun to see if we have random people popping into our standups!

Reply:
Indeed. The _moment_ a _single_ time-waster appears on our daily standup we will be _forced_ to make the link private/invite-only. But until that happens we can be _open_ with it. 👍